homeassistant/components/media_player/yamaha.py

```diff
@@ -39,12 +41,20 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     import rxv
 
     name = config.get(CONF_NAME)
+    host = config.get(CONF_HOST)
     source_ignore = config.get(CONF_SOURCE_IGNORE)
     source_names = config.get(CONF_SOURCE_NAMES)
 
-    add_devices(
-        YamahaDevice(name, receiver, source_ignore, source_names)
-        for receiver in rxv.find())
+    if host == 'DISCO':
+        add_devices(
+            YamahaDevice(name, receiver, source_ignore, source_names)
+            for receiver in rxv.find())
+    else:
+        receivers = \
+            [rxv.RXV("http://"+host+":80/YamahaRemoteControl/ctrl", name)]
```

Comment:
I have implemented string formatting in https://github.com/home-assistant/home-assistant/pull/3451/commits/f879a173df756101450de632fda7ccd6e3c1ae82